src/CartingService/Program.cs

```diff
@@ -45,6 +49,8 @@
 
 builder.Services.ConfigureOptions<ConfigureSwaggerOptions>();
 
+CheckBusSettings(builder);
```

Comment:
What is the point of this method?
Please, check this article for the ideas about validation of appsettings https://code-maze.com/aspnet-configuration-options-validation/

Reply:
Actually, that was only for me. Just to check if it works :)